src/main/resources/hudson/plugins/ec2/SlaveTemplate/help-connectBySSHProcess.html

```diff
@@ -0,0 +1,34 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number of other of contributors
```

Comment:
I think this license header is wrong. You author of this file, but you define that it was create by @kohsuke that works in Sun microsystem in 2004.